patches/api/0085-Add-PlayerAdvancementCriterionGrantEvent.patch

```diff
@@ -21,17 +23,21 @@ index 0000000000000000000000000000000000000000..bb8d7c959cdea4b66455a49e74804ea4
 +
 +/**
 + * Called when a player is granted a criteria in an advancement.
++ *
++ * Unlike most other events, this happens after the criterion is granted, and is rolled back if canceled
```

Comment:
I think better wording would be something like: Called after a player is granted a criterion in an advancement. If cancelled the criteria will be revoked.

Reply:
I agree that this is better wording.